_docs/_latest/develop/test/integration-testing/create-test.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Create Tests
+order: 044
+---
+
+# {{ page.title }}
+
+This page describes how to add integration tests for your Forseti contributions.
+
+---
+## **Overview**
+
+The tech stack used to run Forseti integration tests consists of Kitchen, Kitchen 
+Terraform, InSpec and Google Cloud Build.
+
+[Kitchen](https://kitchen.ci/) provides a test harness to execute infrastructure 
+code on one or more platforms in isolation. It supports many testing frameworks 
+out of the box including InSpec.
+
+[Kitchen-Terraform](https://github.com/newcontext-oss/kitchen-terraform) is an 
+open source set of Test-Kitchen plugins for testing Terraform configuration. It 
+is a tool that lets us test Terraform modules using InSpec. 
+
+[InSpec](https://www.inspec.io/) is a free and open-source framework for testing 
+and auditing applications and infrastructure. InSpec works by comparing the 
+actual state of the system with the desired state expressed in InSpec code. It 
+detects violations and displays findings in the form of a report, but puts us in 
+control of remediation.
+
+[Cloud Build](https://cloud.google.com/cloud-build/docs/) is a service that 
+executes your builds on Google Cloud Platform infrastructure. Cloud Build can 
+import source code from Google Cloud Storage, Cloud Source Repositories, GitHub, 
+or Bitbucket, execute a build to your specifications, and produce artifacts such 
+as Docker containers or Java archives. 
+
+---
+
+Integration tests are hosted in the [forseti-security repository](https://github.com/forseti-security/forseti-security/tree/master/integration_tests/tests/forseti). 
+InSpec framework is used to write integration tests.
+
+## **Adding an integration test**
+
+- Integration tests are enclosed in a `describe` block. Multiple `describe`
```

Comment:
Similar to above comment, I would prefer that we not add any more InSpec tests. Can we provide instructions for creating tests with pytest?

Reply:
Majority of integration tests today are written using InSpec framework. Also, #3578 and the OKR is about moving existing documentation (uses InSpec framework) to the website. I can create an issue to add tests with pytest, and remove instructions to add InSpec tests.